src/core/etl/tests/Flow/ETL/Tests/Unit/DataFrameTest.php

```diff
@@ -355,12 +355,6 @@ public function extract(FlowContext $context) : \Generator
                                     new IntegerEntry('item-id', 1),
                                     new StringEntry('name', 'one'),
                                 ),
-                                new Row\Entry\CollectionEntry(
-                                    'tags',
-                                    new Row\Entries(new IntegerEntry('item-id', 1), new StringEntry('name', 'one')),
-                                    new Row\Entries(new IntegerEntry('item-id', 2), new StringEntry('name', 'two')),
-                                    new Row\Entries(new IntegerEntry('item-id', 3), new StringEntry('name', 'three'))
-                                ),
```

Comment:
This should be replaced MapType or StructType

Reply:
We don't have map yet, but the structure is already covered in the test (above).